esy-package-config/Resolution.re

```diff
@@ -5,7 +5,10 @@ type t = {
   resolution,
 }
 and resolution =
-  | Version(Version.t)
+  | VersionOverride({
+      version: Version.t,
+      override: option(Json.t),
+    })
   | SourceOverride({
       source: Source.t,
       override: Json.t,
```

Comment:
I thought about adding a `SourceAndVersionOverride` but it's implications are not clear to me